docs/sourcing/private-trackers.md

```diff
@@ -1,35 +1,49 @@
 ---
-label: Private BitTorrent Trackers
+label: Private Trackers
 order: -3
 ---
 
-# Private BitTorrent Trackers
+# Private Trackers
 
-## Private Trackers
+Private trackers are BitTorrent trackers that require user registration in order to access and download torrents.
 
-- [AnimeBytes](https://animebytes.tv/)
-- [BakaBT](http://bakabt.me/)
-- [AnimeTorrents](https://animetorrents.me/)
+If you're new to torrenting, see the [torrenting guide](/getting-started/torrenting) on how to get started. *Additionally, we suggest you stick to [public trackers](/sourcing/public-trackers), as most shows can be found there without being locked behind user signup.*
+
+- [AnimeBytes](https://animebytes.tv)
+- [AnimeTorrents](https://animetorrents.me)
+- [BakaBT](https://bakabt.me)
+- [Skyey Snow](https://skyeysnow.com) [!badge variant="success" icon="sign-in" text="Open Signup"]
 - [U2](https://u2.dmhy.org)
-- [Skyeysnow](https://skyeysnow.com/) (open signup)
 
-## How do I join private trackers?
+## AnimeBytes
+
+[AnimeBytes](https://animebytes.tv) (AB) is a closed private BitTorrent tracker for anime.
+
+Users are can join this tracker through recruitments on various top-tier trackers. Unlike other trackers, joining AB is difficult and *not* guaranteed, as several factors can affect your standing, such as receiving bans from other trackers. Below are some examples of ways to be eligible:
+
+- Power User at [RED](https://interviewfor.red) for 12 months
+- Elite Gamer at [GGn](https://gazellegames.net) for 3 months
```

Comment:
publicly mentioning routes is against tracker rules so remove it.

Reply:
This was initially carried over from the original page. Resolved in https://github.com/Snaacky/thewiki/pull/114/commits/994ff16b0a8c3f3d75903607b032f75d4b84e2c1.

Reply:
yep, went unnoticed for ages.